pkg/server/commands/read_changes.go

```diff
@@ -63,8 +65,26 @@ func (q *ReadChangesQuery) Execute(ctx context.Context, req *openfgav1.ReadChang
 	if err != nil {
 		return nil, serverErrors.InvalidContinuationToken
 	}
+	token := string(decodedContToken)
+
+	var startTime time.Time
+	if req.GetStartTime() != nil {
+		startTime = req.GetStartTime().AsTime()
+	}
+	if token == "" && !startTime.IsZero() {
+		tokenUlid := ulid.MustNew(ulid.Timestamp(startTime), nil).String()
```

Comment:
Are there any guarantees that this won't fail? As in this is a user provided value, and yes there is validation layer, but using `MustNew` rather than `New` seems a bit insecure. I'd argue that `MustNew` should only be used in tests.

Reply:
Good call, much safer to use New and return an error if it fails.